src/Microsoft.DotNet.XHarness.Tests.Runners/ApplicationOptions.cs

```diff
@@ -82,6 +84,12 @@ public ApplicationOptions()
                 { "xmlversion", "The xml version.", v => XmlVersion = (XmlVersion) Enum.Parse (typeof (XmlVersion), v, false) },
                 { "logfile=", "A path where output will be saved.", v => LogFile = v },
                 { "result=", "The path to be used to store the result", v => ResultFile = v},
+                { "run-all-tests:", "Run all the tests fund in the assembly. Defaults to true.", v =>
```

Comment:
```suggestion
                { "run-all-tests:", "Run all the tests found in the assembly. Defaults to true.", v =>
```

The `:` belongs here, I know 😁